MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock

```diff
@@ -19,8 +19,8 @@ PODS:
     - MapboxMobileEvents (~> 0.10.2)
     - MapboxSpeech (~> 1.0)
     - Solar (~> 2.1)
-  - MapboxNavigationNative (22.0.5):
-    - MapboxCommon (= 7.1.2)
+  - MapboxNavigationNative (25.1.0):
+    - MapboxCommon (= 7.1.1)
```

Comment:
Oddly, NN v25 uses older `MapboxCommon` version than v22. Could this be a reason for CI Pod check? Verbose output shows an issue with linking `MapboxCommon` slices:
```
-> MapboxCoreNavigation (1.2.0-alpha.1)
    - ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code.
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | xcodebuild:  note: Building targets in parallel
    - NOTE  | xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | [iOS] xcodebuild:  note: Planning build
    - NOTE  | [iOS] xcodebuild:  note: Constructing build description
    - NOTE  | [iOS] xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')
    - NOTE  | [iOS] xcodebuild:  warning: [CP] Unable to find matching .xcframework slice in 'MapboxCommon/MapboxCommon.xcframework MapboxCommon framework ios-i386_x86_64-simulator ios-armv7_arm64 ios-x86_64-maccatalyst' for the current build architectures (arm64 x86_64 i386).
    - NOTE  | xcodebuild:  ld: warning: directory not found for option '-F/Users/viktorkononov/Library/Developer/Xcode/DerivedData/App-dxvmejvrbgstrehborjnnbpvaujh/Build/Products/Release-iphonesimulator/XCFrameworkIntermediates/MapboxCommon'
    - NOTE  | xcodebuild:  clang: error: linker command failed with exit code 1 (use -v to see invocation)
```